### ISSUE TITLE
EO no longer works after HTML5 update

### ISSUE THREAD

**Issue by leeyc0** [andanteyk/ElectronicObserver#229]
The browser EO currently using is IE-based, which no longer works in HTML5 version.

**Comment by leeyc0** [comment]
Also, the API seem completely changed too. KC3改 cannot process the API

**Comment by RadarNyan** [comment]
I'm working on a Chromium Based EOBrowser replacement.

https://github.com/RadarNyan/CefEOBrowser
Currently I have a basic working prototype, you can try grab it from appveyor ci (links in release page)

**Comment by reniris** [comment]
I  wrote #232 , #233 
I want to merge your project

**Comment by RadarNyan** [comment]
@reniris I don't think it's a good idea to merge now as it's no where near a "stable" release yet.

I'm having trouble nailing down the cause of random freeze of 74EO after replacing the browser. Although 74EO freezes, CefEOBrowser keeps running just fine (except I can't get input to it), any help would be appreciated.

Also I'm generally having trouble debugging this project due to lack of memory... just rush ordered a new ram stick and hopefully I can get it in the next few days.

**Comment by reniris** [comment]
There is no confirmation
The compatibility between Cefsharp and Nekoxy is bad

**Comment by ForsakenRei** [comment]
I agree with the opinion about waiting longer for a stable release.  C2 is struggling with the RAM consuming because not just 74EO even native Chrome/Edge suffer memory leaking now. Now I think the Cef alternative is fairly enough for Summer(Fall?) Event in next month.

@RadarNyan I didn't encounter any freeze but sometimes the Cef browser will become complete white and I need to relaunch 74EO. I'm not sure whether it is caused by Cef or Windows because [#211](https://github.com/andanteyk/ElectronicObserver/issues/211) exists before Phase 2.

**Comment by RadarNyan** [comment]
@ForsakenRei I just experienced the same issue, it seems has something to do with ```http://rt.gsspat.jp/g?rtid=13973&j=1```, maybe it's the same issue as https://github.com/poooi/poi/issues/1721

**Comment by andanteyk** [comment]
返信が遅くなり申し訳ありません。

e488ca566eb6320ff14b28e2c95eb6edfe42b290 でブラウザを CefSharp に換装しました。
変更点・既知の問題点はコミットメッセージを参照してください。

簡単にテストした範囲では、概ね以前と同じように動作しています。
何か問題点・不具合・明らかな動作の違いなどがあればご報告いただけると助かります。

なお、ブラウザ以外の問題(「随伴艦一括解除」が正しく反映されない問題等)は別に修正します。
